### DISCUSSION TITLE
Which documentation tool is used by mmdet

### DISCUSSION THREAD

**Discussion by sanchit2843** [open-mmlab/mmdetection#9057] [Q&A] [answered]
### 📚 The doc issue

This is not an issue, I was reading about documenting python code and really like the documentation format of mmdet and pytorch. Any ideas which tool is used to generate the documentation for the code? I have read about sphinx, is this sphinx only, if not which tool is used?

**Comment by ZwwWayne** [comment, marked as answer]
Based on sphinx, we also use the theme in https://github.com/open-mmlab/pytorch_sphinx_theme
